game_app/menus/tic_tac_toe_launch_menu/tic_tac_toe_launch_menu_logic.py

```diff
@@ -24,81 +23,100 @@ def __init__(self, app):
                                                                 False),
                                         RectangularChoiceButton("5",
                                                                 lambda: self.change_marks_required(5),
+                                                                app.settings,
                                                                 (750, 300),
                                                                 (130, 100),
                                                                 False),
                                         ]
 
         self._mark_buttons = [RectangularChoiceButton("X",
                                                       lambda: self.change_mark(0),
+                                                      app.settings,
                                                       (490, 500),
                                                       (130, 100),
                                                       True),
                               RectangularChoiceButton("O",
                                                       lambda: self.change_mark(1),
+                                                      app.settings,
                                                       (660, 500),
                                                       (130, 100),
                                                       False),
                               ]
 
         self._start_button = DisableableRectangularTextButton("Let's go!",
                                                               "Bad params",
-                                                              lambda: self.switch_to_tic_tac_toe(),
+                                                              self.switch_to_tic_tac_toe,
+                                                              app.settings,
                                                               (1030, 590),
                                                               (230, 110),
                                                               False)
 
         self._back_to_menu_button = RoundIconButton(
-            'game_app/resources/images/tic_tac_toe_launch_menu/left_arrow_white.png',
-            lambda: self.switch_back_to_main_menu(),
+            self._resolve_back_arrow_image_path(app.settings[Settings.COLOR]),
+            self.switch_back_to_main_menu,
+            app.settings,
             (40, 40),
             30)
 
         self.all_buttons = [self._start_button, self._back_to_menu_button] + \
                            self._size_buttons + self._marks_required_buttons + self._mark_buttons
 
+    def _resolve_back_arrow_image_path(self, color_mode):
+        if color_mode == ColorMode.DARK:
+            return 'game_app/resources/images/common/left_arrow_white.png'
+        else:
+            return 'game_app/resources/images/common/left_arrow_black.png'
+
```

Comment:
Could be a little bit more dry

Reply:
rzeczywiscie, zaraz poprawie tu i w settings_logic jest masa takich xd